### DISCUSSION TITLE
No blocking_send on Broadcast channel?

### DISCUSSION THREAD

**Discussion by xanderdunn** [tokio-rs/tokio#4752] [Q&A] [answered]
`mpsc::Sender` has a [blocking_send](https://docs.rs/tokio/latest/tokio/sync/mpsc/struct.Sender.html#method.blocking_send) for synchronization between blocking and async tasks. I don't see any similar functionality for the broadcast channel. I have a channel that needs to receive messages from multiple sources, send those messages to multiple destinations, and some of the senders are blocking tasks because they're completing computationally intensive work. Is this a feature that just hasn't been implemented yet, or is there a fundamental reason this can't/shouldn't be done with a broadcast channel?

Thanks!

**Comment by Darksonn** [comment, marked as answer]
It just has not been implemented yet.
